app/src/main/java/com/rlab/sejima/features/MUTopBar.java

```diff
@@ -101,25 +104,29 @@ public MUTopBar(Context context) {
     public MUTopBar(Context context, AttributeSet attrs) {
         super(context, attrs);
 
-        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.MUTopBar);
+        TypedArray attributes = context.obtainStyledAttributes(attrs, R.styleable.MUTopBar);
         CharSequence s;
 
         // Deal with title's attributes
-        s = a.getString(R.styleable.MUTopBar_title);
+        s = attributes.getString(R.styleable.MUTopBar_title);
         mTitle = TextUtils.isEmpty(s) ? mTitle : s.toString();
-        mTitleColor = a.getColor(R.styleable.MUTopBar_title_color, mTitleColor);
-        mTitleFontSize = a.getDimensionPixelSize(R.styleable.MUTopBar_title_size, 0);
-        mTitleFontWeight = a.getInt(R.styleable.MUTopBar_title_weight, mTitleFontWeight);
-        mTitleAlignment = a.getInt(R.styleable.MUTopBar_topbar_title_alignment, mTitleAlignment);
+        mTitleColor = attributes.getColor(R.styleable.MUTopBar_title_color, mTitleColor);
+        mTitleFontSize = attributes.getDimensionPixelSize(R.styleable.MUTopBar_title_size, 0);
+        mTitleFontWeight = attributes.getInt(R.styleable.MUTopBar_title_weight, mTitleFontWeight);
+        mTitleAlignment = attributes.getInt(R.styleable.MUTopBar_topbar_title_alignment, mTitleAlignment);
 
-        mLeftButtonWidth = a.getDimensionPixelSize(R.styleable.MUTopBar_topbar_img_width, 0);
-        mLeftButtonLeading = a.getDimensionPixelSize(R.styleable.MUTopBar_topbar_btn_leading, 0);
-        mButtonImage = a.getResourceId(R.styleable.MUTopBar_topbar_btn_img, mButtonImage);
+        mLeftButtonWidth = attributes.getDimensionPixelSize(R.styleable.MUTopBar_topbar_img_width, 0);
+        mLeftButtonLeading = attributes.getDimensionPixelSize(R.styleable.MUTopBar_topbar_btn_leading, 0);
+        mButtonImage = attributes.getResourceId(R.styleable.MUTopBar_topbar_btn_img, mButtonImage);
 
         init(context);
-        a.recycle();
+        attributes.recycle();
     }
 
+    /**
+     *
+     * @return the left padding of image button
+     */
     public float getLeftButtonLeading() {
```

Comment:
Done for getLeftButtonLeading but still need documentation for:
- isButtonHidden()
- setButtonHidden(boolean hideButton)
- getMUTopBarClickListener()
- setMUTopBarClickListener(MUTopBarClickListener clickListener)